swoloszynski/journey-planner
# Issue 29 add linter, closes #29

### What
- A "linter" is a type of software/library that will automatically check our code for style consistency and keep things clean.
- Added [JS Hint](http://jshint.com/docs/) 
- Added JS Hint default configs in the .jshintrc file
- Customized a few settings in the config file (separated, at the top of the file)
- Added an npm command for linting all code files (app.js, all src and test files) - `npm run lint`
- Included linting in test command `npm test`
- Cleans up the files to resolve any linting complaints

### How to test
- Remove a semi colon or otherwise break the rules, then run `npm test` and `npm run lint` to see that it highlights the error. Change it back, rerun, and see that the linter is happy.
